c_src/sqlite3_nif.c

```diff
@@ -638,6 +638,135 @@ exqlite_bind(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
     return make_atom(env, "ok");
 }
 
+static ERL_NIF_TERM
+exqlite_bind_text(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
+{
+    assert(argc == 3);
```

Comment:
When this is compiled in release mode, these asserts disappear. I think we need to do a check and raise immediately.

```c
if (argc != 3) {
  ERL_NIF_TERM badarg = make_message(env, "invalid arguments");
  return enif_raise_exception(env, badarg);
}
```

Reply:
I think it's OK. We only define /3 function.

Reply:
True. Just my crazy side of not wanting to crash the erlang VM because someone got wise and invoked it as a `/2` somehow.

Reply:
Yea just ignore me here, I can't think of a way to invoke this from erlang or elixir in a manner that gives us too many args or not enough.